docs/content/testing/testing-guidelines.md

```diff
@@ -0,0 +1,252 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing guidelines
```

Comment:
See: https://docs.google.com/document/d/1TfVlBR0_zENPniIKNDjXYlHJPnFB-_ltqPoHnK-6Bng/edit

Reply:
(Thought using a GDoc is less painful.)